intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_utils.cpp

```diff
@@ -926,7 +926,7 @@ size_t jblas_quantize(const float* f32ptr, void* dstpr, const quant_params_inter
         using KernelRef = WeiS8Fp32<GcCompFp32, JblasNoSIMD>;
         static Kernel kernel;
         static KernelRef kernelref;
-        auto packedw = kernel.createStorage(n, k, params.group_size, params.alg == quant_alg::sym);
+        auto packedw = kernel.createStorage(n, k, params.group_size, params.alg == quant_alg::asym);
```

Comment:
why change sym to asym ? 
the function is still same ?

Reply:
This code is wrong before. This PR fix it.